### ISSUE TITLE
Any plans for Hibernate 4.x or JPA?

### ISSUE THREAD

**Issue by mhellkamp** [jirutka/rsql-hibernate#2]
This is a really useful module. Do you have any plans to support Hibernate 4 or JPA? The package structure for the Criteria classes has changed in Hibernate 4 so I have to revert back to 3.x to use the module.

**Comment by rpvilao** [comment]
I'm using this and so far is the best approach to an unsolved problem I've seen. My concern is that the project remains untouched two years ago :/ It is possible to at least release the commons to a final version instead of snapshot? I can probably provide some support. Also, I integrated this with spring where I created annotations for DTO to Entity mappings and type conversions as well. I think that in the future a spring module can be done out of this as well. One more question is: why isn't it in maven central like the rsql-parser one? Again, great module I would like to see this getting somewhere :)
